lib/HttpSignatureStrategy.js

```diff
@@ -78,12 +81,14 @@ Strategy.prototype.authenticate = function(req) {
             requiredHeaders: REQUIRED_HEADERS
           }));
       }
-      callback();
+      callback(null);
```

Comment:
Why did you add `null` here? We usually do the simplest thing, which is to call `callback();` with no params, which will set `err` to undefined. There have been some places in various libs that people have done `callback(null);` but I think those are now proliferating when they should be replaced with `callback();`. If there's not a good reason to pass the extra parameter, don't.